development/kops/set_environment.sh

```diff
@@ -13,9 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-BASEDIR=$(dirname "$0")
 export RELEASE_BRANCH=${RELEASE_BRANCH:-"1-19"}
-export DEFAULT_RELEASE=$(cat ${BASEDIR}/../../release/${RELEASE_BRANCH}/RELEASE)
+export DEFAULT_RELEASE=$(cat ../../release/${RELEASE_BRANCH}/RELEASE|tr -d ' ')
```

Comment:
Yeh, I've been thinking we need another strategy with this, I think we need to not put spaces in the file anymore and use another strategy to start a release.

Reply:
```suggestion
export DEFAULT_RELEASE=$(cat ${BASEDIR}/../../release/${RELEASE_BRANCH}/RELEASE)
```